### PR TITLE
Feature/codacy update

### PR DESCRIPTION
**Pull request by detpircs**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/detpircs/inputCommand/pull/1" target="_blank">ReviewNinja

File: assets/css/app.css [hunk 1]
```diff
@@ -66,14 +66,15 @@ body {
 	border-color: #5A5523;
 }
 input[type="text"] {
-	padding: 1em;
+	padding: 32px;
 	font-size: 2em;
```
Comment (detpircs) [inline assets/css/app.css]: Good work

Comment (detpircs) [inline assets/css/app.css]: fix this? !fix

Comment (detpircs) [inline assets/css/app.css]: fixed it... !fixed

**Pull request merged**